scripts/ansible_v2_3_2_0_1_contrib_inventory_openstack.py

```diff
@@ -119,13 +125,15 @@ def get_host_groups(inventory, refresh=False):
 
 
 def append_hostvars(hostvars, groups, key, server, namegroup=False):
-    # NOTE(mattt): Patched locally until
-    #              https://github.com/ansible/ansible/pull/34452 merges
-    #              and we're able to bump to latest 2.3 release.
     hostvars[key] = dict(
         ansible_ssh_host=server['interface_ip'],
         ansible_host=server['interface_ip'],
         openstack=server)
+
+    metadata = server.get('metadata', {})
```

Comment:
Sure, but instead of trying to pick parts of it I thought it's be simpler to maintain if we just grab the thing wholesale. This is a stop-gap until https://rpc-openstack.atlassian.net/browse/RE-1721 where all this changes and we don't need to vendor this script any more.